docs/developers/docker.rst

```diff
@@ -104,6 +104,11 @@ finished setup and the webservice is running. Then use:
 
     docker exec -it invenio_web_1 python setup.py test
 
+.. NOTE::
```

Comment:
Doesn't matter, sphinx gets both. The rest of the file uses the uppercase version.